From penberg@cs.helsinki.fi  Wed Nov 12 13:52:26 2008
From: Pekka Enberg <penberg@cs.helsinki.fi>
Date: Thu, 30 Oct 2008 18:12:03 +0200
Subject: Staging: w35und: remove unused ->ShutDowned member from struct LOCAL_PARA
To: greg@kroah.com
Cc: Pekka Enberg <penberg@cs.helsinki.fi>, Pavel Machek <pavel@suse.cz>
Message-ID: <1225383127-16406-3-git-send-email-penberg@cs.helsinki.fi>


It's not actually read by anyone so we might as well remove it.

Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/localpara.h |    1 -
 drivers/staging/winbond/wbusb.c     |    6 ------
 2 files changed, 7 deletions(-)

--- a/drivers/staging/winbond/localpara.h
+++ b/drivers/staging/winbond/localpara.h
@@ -149,7 +149,6 @@ typedef struct LOCAL_PARA
 
     //// power-save variables
     u8  		  	iPowerSaveMode;     // 0 indicates it is on, 1 indicates it is off
-	u8			ShutDowned;
 	u8			ATIMmode;
 	u8			ExcludeUnencrypted;
 
--- a/drivers/staging/winbond/wbusb.c
+++ b/drivers/staging/winbond/wbusb.c
@@ -220,9 +220,6 @@ static unsigned char wb35_hw_init(struct
 	pHwData = &priv->sHwData;
 	hal_set_phy_type( pHwData, RF_DECIDE_BY_INF );
 
-	// Initial Software variable
-	priv->sLocalPara.ShutDowned = false;
-
 	//added by ws for wep key error detection
 	priv->sLocalPara.bWepKeyError= false;
 	priv->sLocalPara.bToSelfPacketReceived = false;
@@ -424,9 +421,6 @@ void packet_came(struct ieee80211_hw *hw
 
 static void wb35_hw_halt(struct wbsoft_priv *adapter)
 {
-	//---------------------
-	adapter->sLocalPara.ShutDowned = true;
-
 	Mds_Destroy( adapter );
 
 	// Turn off Rx and Tx hardware ability
